zerver/worker/queue_processors.py

```diff
@@ -427,6 +430,25 @@ def consume(self, event):
 class OutgoingWebhookWorker(QueueProcessingWorker):
     def consume(self, event):
         # type: (Mapping[str, Any]) -> None
-
-        # TODO: Implement this worker.
-        pass
+        message = event['message']
+        realm = get_realm_by_email_domain(message['sender_email'])
+        bot_user = get_user_profile_by_id(event['user_profile_id'])
+        services = get_realm_bot_services(bot_user.email, realm)
+        rest_operation = {'method': 'POST',
+                          'relative_url_path': '',
+                          'request_kwargs': {},
+                          'base_url': ''}
+        trigger_cache = dict()
+        for key in event:
+            trigger_cache[key] = event[key]
+
+        TRIGGER_RE = re.compile(u'(@\*\*.*\*\* )(.*)')
+        if event['trigger'] == "mention":
+            trigger_cache['command'] = str(TRIGGER_RE.search(message['content']).group(2))
+        else:
+            trigger_cache['command'] = message['content']
```

Comment:
Trigger_cache is being mainly used here for retries. It is a copy of the event inserted into the queue. So in case of a retry request, this copy is inserted again into the queue, with the retry count incremented. I think renaming it to 'event' would help avoid all confusions.  
I agree about parsing logic, ideally it should reside inside the interface and for this initial version we can skip it.